qiskit/tools/events/progressbar.py

```diff
@@ -118,6 +118,16 @@ class TextProgressBar(BaseProgressBar):
 
     output_handler : the handler the progress bar should be written to, default
                      is sys.stdout, another option is sys.stderr
+
+    .. jupyter-execute::
+
+       import numpy as np
+       import qiskit.tools.jupyter
+       from qiskit.tools.parallel import parallel_map
+       from qiskit.tools.events import TextProgressBar
+       TextProgressBar()
+       %qiskit_progress_bar -t text
+       parallel_map(np.sin, np.linspace(0,10,100));
```

Comment:
Does the progress bar only work with the `parallel_map`? 🙂 Otherwise it might be nice to keep the example minimal and use an easier workflow.

Reply:
What about an example like
```python
import numpy as np
from qiskit.tools.events import TextProgressBar

iterations = 100
t = TextProgressBar()
t.start(iterations=iterations)

for i in range(iterations):
    # step i of heavy calculation ...
    t.update(i + 1)  # update progress bar
```
🙂